routes/raker.router.js

```diff
@@ -0,0 +1,194 @@
+const express = require('express');
+const {db} = require('../server/db')
+const router = express.Router();
+
+//get rakers
```

Comment:
🚫 **[eslint]** <[spaced-comment](https://eslint.org/docs/rules/spaced-comment)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected exception block, space or tab after '//' in comment.
```suggestion
// get rakers
```